learn/what_is_meilisearch/comparison_to_alternatives.mdx

```diff
@@ -180,6 +180,8 @@ Elasticsearch can sometimes be too slow if you want to provide a full instant se
 
 Meilisearch is a perfect choice if you need a simple and easy tool to deploy a typo-tolerant search bar. It provides prefix searching capability, makes search intuitive for users, and returns results instantly with excellent relevance out of the box.
 
+For a more detailed analysis of how it compares with Meilisearch, refer to our [Elasticsearch comparison page](https://blog.meilisearch.com/meilisearch-vs-elasticsearch/).
```

Comment:
Can you please add `?utm_campaign=oss&utm_source=docs&utm_medium=comparison` at the end of the URL?